lms/templates/dashboard/_dashboard_xseries_info.html

```diff
@@ -0,0 +1,35 @@
+<%page args="course_program_info, enrollment_mode" />
+<%!
+    from django.utils.translation import ugettext as _
+%>
+<%namespace name='static' file='../static_content.html'/>
+<div class="message message-status is-shown credit-message">
+    <div class="xseries-action">
+        <div class="message-copy xseries-msg">
+            <p>
+                <b class="message-copy-bold">${_("{category} Program: Interested in more courses in this subject?").format(category=course_program_info['display_category'])}</b>
+            <p>
+            <p class="message-copy">
+                ${_("This course is 1 of {course_count} courses in the {link_start}{course_display}{link_end} {category}.").format(
```

Comment:
@awais786 this will be easier for a translator to understand if the placeholders read more easily. For example, "This course is 1 of {course_count} courses in the {link_start}{program_display_name}{link_end} {program_category}."